Source/WebCore/platform/graphics/texmap/TextureMapperLayer.cpp

```diff
@@ -76,6 +110,112 @@ TextureMapperLayer::~TextureMapperLayer()
     removeFromParent();
 }
 
+void TextureMapperLayer::preprocess(TextureMapperPaintOptions& options)
```

Comment:
Not clear from the naming what this function does.
Something more descriptive should be used, maybe `prepareLayerForPainting` (not good, just as first suggestion), `flatternLayerBeforePaintingIfNecessary` ?